SledRacing/Include/Style.css

```diff
@@ -68,7 +68,8 @@ body {
 }
 .in-game {
 	-webkit-text-stroke: 1px #0184CE;
-	background-color: black;
+	background-image: url("Assets/Background.png");
```

Comment:
By the way, you'll probably need to double check if this works. It should do, but there's no other examples so I'm hesitant it might not